plat/renesas/rcar/aarch64/rcar_common.c

```diff
@@ -58,6 +58,10 @@ const uint8_t version_of_renesas[VERSION_OF_RENESAS_MAXLEN]
 					DRAM1_NS_SIZE,			\
 					MT_MEMORY | MT_RW | MT_NS)
 
+#define MAP_DRAM1_NS_SCPI	MAP_REGION_FLAT(DRAM1_NS_SCPI_BASE,		\
+					DRAM1_NS_SCPI_SIZE,			\
+					MT_DEVICE | MT_RW | MT_NS)
```

Comment:
Why you are mapping it as `MT_DEVICE`? This should be memory region. Maybe with cache disabled, but memory.

Reply:
Good question. Will correct.